plugin-root/gradle

```diff
@@ -0,0 +1 @@
+../gradle
```

Comment:
I removed this symlink as well, it seems to work without. I added them in case someone wanted to open `gradle-plugin`  from intelliJ but now that the root project is the composite one, it shouldn't matter much.